libs/text-splitters/pyproject.toml

```diff
@@ -12,6 +12,7 @@ repository = "https://github.com/langchain-ai/langchain"
 python = ">=3.8.1,<4.0"
 langchain-core = "^0.1.28"
 lxml = {version = "^5.1.0", optional = true}
+semantic-text-splitter = "^0.6.3"
```

Comment:
@baskaryan what do you think optionals here?

Reply:
Updated as true